mllib/src/test/scala/org/apache/spark/ml/feature/CountVectorizerSuite.scala

```diff
@@ -164,4 +167,23 @@ class CountVectorizerSuite extends SparkFunSuite with MLlibTestSparkContext {
         assert(features ~== expected absTol 1e-14)
     }
   }
+
+  test("CountVectorizer read/write") {
```

Comment:
If the Estimator and Model share the same Params, it could be easier and shorter to use ```testEstimatorAndModelReadWrite```